Userland/Libraries/LibWeb/CodeGenerators/WrapperGenerator.cpp

```diff
@@ -407,7 +407,7 @@ int main(int argc, char** argv)
         return 1;
     }
 
-    if (namespace_.is_one_of("CSS", "DOM", "HTML", "UIEvents", "HighResolutionTime", "NavigationTiming", "SVG", "XHR")) {
+    if (namespace_.is_one_of("Communication", "CSS", "DOM", "HTML", "UIEvents", "HighResolutionTime", "NavigationTiming", "SVG", "WebSocket", "XHR")) {
```

Comment:
There are no `Communication` or `WebSocket` spec that I'm aware of(?)
WebSocket is part of the HTML spec, isn't it?

Reply:
`WebSocket` is indeed defined as part of the HTML living standard, under `section 9: Communication` (https://html.spec.whatwg.org/multipage/#toc-comms). I'll move it into the HTML folder directly then, that makes more sense :)